src/ConfirmationTooltip/index.tsx

```diff
@@ -32,28 +36,39 @@ export const ConfirmationTooltip: React.FC<Props> = ({
     };
   }, []);
 
-  return (
-    <AbstractTooltip
-      trigger="click"
-      {...props}
-      onShow={(instance) => {
-        if (timeoutRef.current) {
-          window.clearTimeout(timeoutRef.current);
-        }
-
-        timeoutRef.current = window.setTimeout(() => {
-          instance.hide();
-        }, 3000);
-      }}
-      onHide={() => {
-        if (timeoutRef.current) {
-          window.clearTimeout(timeoutRef.current);
-        }
+  const [displayedContent, setDisplayedContent] = React.useState(
+    initalContent ?? content,
+  );
 
-        timeoutRef.current = null;
+  return (
+    <div
```

Comment:
might adding a div wrapper cause unexpected layout problems?  

(my understanding is that this component did not previously generate DOM elements)?

Ideally if possible we could make our components headless (in the vein of downshift and headless-ui)